### PR TITLE
Sprint 9 - Request to merge Dashboard into Dev

### PR DESCRIPTION
**Pull request by arsanig**
Sprint 9 - Request to merge Dashboard into Dev
Functionalities Added: 
   - New benefit cards and application cards
   - New icons
   - Updated tests with Accessibility testing

### CODE REVIEW

**Review by DaceyTom2 — COMMENTED** [review]
Looks really good! I'm gonna get a second pair of eyes for the approval

**Review by nibivi77 — APPROVED** [review]
This looks good to me guys! Good work.

Only one comment, on Mobile at 320px width, it could use a little work to make it look better. However, I don't know the design. Good work!

**Review by DaceyTom2 — APPROVED** [review]

**Pull request merged**
